tests/dryrun/v3_install_test.go

```diff
@@ -1583,16 +1655,22 @@ var (
 func setupV3Test(t *testing.T, hcli helm.Client, preflightRunner preflights.PreflightRunnerInterface) (string, string) {
 	t.Helper()
 
+	return setupV3TestWithClusterConfig(t, hcli, preflightRunner, clusterConfigData)
```

Comment:
nit: might be better to make `setupV3Test` function accept "options" so we can override more than just the cluster config later if needed. but this is fine for now i guess.

Reply:
done